### PR TITLE
update pay model

### PR DESCRIPTION
**Pull request by michael-rob-poole**
### Description
update pay model

Add a link to the relevant story in Jira
[SASS-884](https://jira.tools.tax.service.gov.uk/browse/SASS-864)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [ ]  Have you checked the PR Builder passes?
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: app/models/employment/Pay.scala [hunk 1]
```diff
@@ -21,8 +21,8 @@ import play.api.libs.json.{Json, OFormat}
 case class Pay(taxablePayToDate: BigDecimal,
                totalTaxToDate: BigDecimal,
                tipsAndOtherPayments: Option[BigDecimal],
```
Comment (ina-tu) [inline app/models/employment/Pay.scala:23]: can you set this to equal 'None'

File: app/models/employment/Pay.scala [hunk 2]
```diff
@@ -21,8 +21,8 @@ import play.api.libs.json.{Json, OFormat}
...
-               payFrequency: String,
-               paymentDate: String,
+               payFrequency: Option[String],
+               paymentDate: Option[String],
                taxWeekNo: Option[Int],
```
Comment (ina-tu) [inline app/models/employment/Pay.scala:26]: can you set this to equal 'None'

File: app/models/employment/Pay.scala [hunk 3]
```diff
@@ -21,8 +21,8 @@ import play.api.libs.json.{Json, OFormat}
...
                taxMonthNo: Option[Int])
```
Comment (ina-tu) [inline app/models/employment/Pay.scala:27]: can you set this to equal 'None'

**Review by ina-tu — CHANGES_REQUESTED** [review]

File: app/models/employment/Pay.scala (hunk 1 above)
Comment (michael-rob-poole) [reply app/models/employment/Pay.scala:23]: Yep - it isn't required by Des.

File: app/models/employment/Pay.scala (hunk 2 above)
Comment (michael-rob-poole) [reply app/models/employment/Pay.scala:26]: same

**Review by ina-tu — APPROVED** [review]

**Pull request merged**